### DISCUSSION TITLE
Changing annotation area font size and line spacing

### DISCUSSION THREAD

**Discussion by alpha-1-centauri** [doccano/doccano#1976] [Q&A]
Hi, 
I am reviewing pretty dense documents and was wondering if there is any way the front end can be modified?
Reducing the font size and spacing would greatly help! Apologies if this is something obvious - I am only just getting started with Doccano and Docker.

Best wishes,

Adam

**Comment by Hironsan** [comment]
For now, there is no way to change font size and spacing. But, I don't think it would be that difficult to implement (since only the front end needs to be changed). If there are many requests, we will consider implementing the feature.

**Reply by alpha-1-centauri** [reply]
I think customizable annotation frontend would really elevate this platform. For now I will resort to scrolling. Thanks for letting me know!

**Comment by david-engelmann** [comment]
This is a request that could be paired with the general push for custom formatting, such as bold or highlight within the text area. I remember seeing an issue related to it

**Comment by aCampello** [comment]
I'm endorsing this issue - it would be great to have more control over formatting aspects such as font-size and line spacing.

**Comment by plegoff** [comment]
I 'm trying Doccano and I agree with this proposition : having more control on the annotation/label police and size willbe helpfull
